source/_integrations/modbus.markdown

```diff
@@ -466,6 +526,12 @@ climates:
 | ------- | ----------- |
 | set_temperature | Set Temperature. Requires `value` to be passed in, which is the desired target temperature. `value` should be in the same type as `data_type` |
 
+### Service `modbus.set-hvac-mode`
+
+| Service | Description |
+| ------- | ----------- |
+| set_hvac_mode | Set HVAC mode. Requires `value` to be passed in, which is the desired mode. `value` should be a valid HVAC mode. A mapping between the desired state and the value to be written to the HVAC mode register must exist. Calling this service will also set the On/Off register to an appropriate value, if such a register is defined. |
+
```

Comment:
How does a user set-hvac-mode to off ?

Reply:
It's a mode like any other, and it works.